usr/share/rear/verify/TSM/default/389_check_TSM_connexion.sh

```diff
@@ -0,0 +1,4 @@
+# Use a first connection to the TSM server to request TSM PASSWD in case this one was not included in the
+# ReaR rescue image.
+# Note: ReaR uses fd6 for user input, fd7 for stdout and fd8 for stderr.
+dsmc query mgmt 0<&6 1>&7 2>&8
```

Comment:
@schabrolles 
the GitHub https://github.com/rear/rear/pull/1641/files
shows a "no newline at end of file" symbol here.
As far as I remember under Unix/Linux text files must have a
final newline, otherwise arbitrary strange issues could happen
I guess in particular tools that read such files line by line
may not read the last line when ther is no final newline.

Reply:
@jsmeix, Ok it is done now